Zend/zend_API.h

```diff
@@ -361,6 +361,7 @@ ZEND_API zend_module_entry* zend_register_module_ex(zend_module_entry *module);
 ZEND_API zend_result zend_startup_module_ex(zend_module_entry *module);
 ZEND_API void zend_startup_modules(void);
 ZEND_API void zend_collect_module_handlers(void);
+ZEND_API void zend_destroy_module_handlers(void);
```

Comment:
I'm not sure, if it's a good idea to change public API in old PHP versions.

Reply:
I removed this function as suggested below.